client/src/graphics/draw.rs

```diff
@@ -461,10 +461,7 @@ impl Draw {
             Uniforms {
                 view_projection,
                 inverse_projection: projection.try_inverse().unwrap(),
-                // Only 1e-2 of color should be visible from view_distance, assuming
-                // exponential-squared fog
-                fog_density: ((1.0f32 / 1e-2).ln().sqrt()
-                    / self.cfg.local_simulation.view_distance),
+                fog_density: (1.0 / self.cfg.local_simulation.view_distance),
```

Comment:
The original math is specifically tuned to (mostly) hide stuff outside of view distance, to mask streaming. It should not be replaced with an ad-hoc factor.

Reply:
Changed it back to the original.